src/main/java/seedu/mindmymoney/command/AddCommand.java

```diff
@@ -147,6 +150,44 @@ public void addExpenditure() throws MindMyMoneyException {
         System.out.print(System.lineSeparator());
     }
 
+    /**
+     * Checks if parsed date is after the current date.
+     *
+     * @param date date that is parsed in.
+     * @throws MindMyMoneyException throws an exception when the date parsed is after current date.
+     */
+    public static void checkAfterCurrentDate(LocalDate date) throws MindMyMoneyException {
```

Comment:
Maybe can consider making this function a boolean instead?

Reply:
Hmm, I did that previously but I realised there will be alot of if statements, hence I decided to just throw the exception in the function itself.